.circleci/config.yml

```diff
@@ -10,16 +10,13 @@ workflows:
       - codacy/checkout_and_version:
           write_sbt_version: true
       - codacy/sbt:
-          name: publish_docker_local
+          name: build_docs_and_docker
           cmd: |
             wget https://github.com/phpDocumentor/phpDocumentor/releases/download/v3.4.3/phpDocumentor.phar
             sudo mv phpDocumentor.phar /usr/local/bin/phpdoc
             sudo chmod +x /usr/local/bin/phpdoc
 
-            sbt "set Docker / version := \"latest\";
-                 runMain codacy.codesniffer.docsgen.GeneratorMain;
-                 clean;
-                 Docker / publishLocal"
+            sbt "runMain codacy.codesniffer.docsgen.GeneratorMain"
```

Comment:
don't you wanna move this to the Dockerfile as well, and eliminate the need for sbt dependency in the CI?

Reply:
Didn't want to force doc generation every time a local build is done but I can work with a ENV to skip doc generation.

Reply:
depending on where it is on the Dockerfile, it'll be cached and won't happen everytime